CHANGELOG.md

```diff
@@ -48,25 +48,25 @@
 
 * Incorporate newer libpg_query updates in 10-1.0.3 and 10-1.0.4
   * Adds support for running on ARM
-  * Fixes asprintf warning during builds
+  * Fixes a sprintf warning during builds
```

Comment:
This is actually intended to reference `asprintf` (the C method)

Reply:
Thanks, fixed!